trade-finance-manager-api/api-tests/v1/deals/deal-controller.api-test.js

```diff
@@ -1,52 +1,35 @@
 const { findOneTfmDeal, findOnePortalDeal } = require('../../../src/v1/controllers/deal.controller');
 const mapDeal = require('../../../src/v1/mappings/map-deal');
 const MOCK_DEAL = require('../../../src/v1/__mocks__/mock-deal');
-const { mockFindOneDeal, mockUpdateDeal } = require('../../../src/v1/__mocks__/common-api-mocks');
+const { mockFindOneDeal } = require('../../../src/v1/__mocks__/common-api-mocks');
 const api = require('../../../src/v1/api');
-const { dealsLightReducer } = require('../../../src/v1/rest-mappings/deals-light');
-const mapDeals = require('../../../src/v1/mappings/map-deals');
-const acbsController = require('../../../src/v1/controllers/acbs.controller');
-
-jest.mock('../../../src/v1/rest-mappings/deals-light');
-jest.mock('../../../src/v1/mappings/map-deals');
-
-const mappedDeal = mapDeal(MOCK_DEAL);
 
 describe('deal controller', () => {
-  acbsController.createACBS = jest.fn();
-
   beforeEach(() => {
-    acbsController.createACBS.mockReset();
-
     api.findOneDeal.mockReset();
-    mockFindOneDeal();
 
```

Comment:
This test file asks a few more questions than it answers...It's mocking `api.findOneDeal` but isn't mocking other api calls that occur -- so it is relying on other services...

For instance
```
api.findOnePortalDeal
api.findOneFacility
api.findFacilitiesByDealId
```

Not for this PR, but this is not following good api test practices (the functions themselves are also rather legacy, and I think they could be improved -- again this is more notes for myself)

Reply:
Interesting - I just removed them as they seemed irrelevant in the test.